README.md

```diff
@@ -92,7 +92,7 @@ To simply run the bot process:
 
 1. Clone this repository.
 
-2. Install Node and `npm`. The instructions [at the NPM website][npm] should suffice.
+2. Install Node v22.14.0 and `npm`. The instructions [at the NPM website][npm] should suffice.
```

Comment:
Maybe say "... and the corresponding version of `npm`" ?

Reply:
Sounds good.  Thanks.

This change is from PR #145 so I've applied it there.